redis/connection.py

```diff
@@ -94,16 +94,16 @@
 
 
 class Encoder(object):
-    "Encode strings to bytes and decode bytes to strings"
+    "Encode strings to bytes-like and decode bytes-like to strings"
 
     def __init__(self, encoding, encoding_errors, decode_responses):
         self.encoding = encoding
         self.encoding_errors = encoding_errors
         self.decode_responses = decode_responses
 
     def encode(self, value):
-        "Return a bytestring representation of the value"
-        if isinstance(value, bytes):
+        "Return a bytestring or bytes-like representation of the value"
+        if isinstance(value, bytes) or isinstance(value, memoryview):
```

Comment:
You can pass multiple types to `isinstance`

```suggestion
        if isinstance(value, (bytes, memoryview)):
```

Reply:
@jdufresne's suggestion makes sense here.